QUARKUS-1398.md

```diff
@@ -0,0 +1,34 @@
+# QUARKUS-1398 - Test and verify GraphQL support
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-1398
+
+Coverage should ensure full GraphQL support from client and server side.
+For native mode similar test coverage as for JVM mode needs to be ensured.
+
+## Scope of the testing
+Quarkus supports GraphQL for a long time, but now this support become official, following actions are needed to ensure test coverage:
```

Comment:
We have had tech support previously, and with 2.7, we are moving towards full support. Please clarify this in the statement.

Reply:
Statement clarified, coverage described.